pkg/agent/watcher.go

```diff
@@ -21,16 +21,18 @@ import (
 	"time"
 
 	core_v1 "k8s.io/api/core/v1"
-	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/watch"
+	//meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8swatch "k8s.io/apimachinery/pkg/watch"
 	k8sclient "k8s.io/client-go/kubernetes"
 
 	"github.com/intel/cri-resource-manager/pkg/log"
 )
 
 type cachedConfig struct {
 	sync.RWMutex
-	config resmgrConfig
```

Comment:
Related to the longer comment below, should we have `defaultCfg` here, too(?)

Reply:
I was thinking about that for cases where we want a different node-local default than the built-in one. However, in other PR's we've been taking out bells and whistles that we were not using at the moment, I decided to omit it and add it only later if it becomes 100 % sure we'll both want and need it.